library/Icingadb/Model/Behavior/ActionAndNoteUrl.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/* Icinga DB Web | (c) 2021 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Model\Behavior;
+
+use ipl\Orm\Contract\PropertyBehavior;
+
+class ActionAndNoteUrl extends PropertyBehavior
+{
+    public function fromDb($value, $key, $_)
+    {
+        if (empty($value)) {
+            return [];
+        }
+
+        $links = [];
+        if (strpos($value, "' ") === false) {
+            $links[] = $value;
+        } else {
+            foreach (explode("' ", $value) as $url) {
+                $url = strpos($url, "'") === 0 ? substr($url, 1) : $url;
+                $url = strpos($url, "'") === strlen($url) - 1 ? substr($url, 0, strlen($url) - 1) : $url;
+                $links[] = $url;
+            }
+        }
+
+        return $links;
+    }
+
+    public function toDb($value, $key, $_)
+    {
+        if (empty($value)) {
+            return $value;
+        }
+
+        $links = '';
+        foreach ($value as $url) {
+            if (! empty($links)) {
+                $links .= ' ';
+            }
+            $links .= $url;
```

Comment:
See https://github.com/Icinga/icingadb-web/pull/303#discussion_r700866495

Reply:
Quoting because it's identified as identifier by the database otherwise is one thing. The quoting I'm talking of above is the format used to designate multiple urls in the configuration option for `action_url` and `notes_url`. Take a look at the parsing algorithm.